wp-admin/css/colors/_admin.scss

```diff
@@ -431,11 +433,13 @@ ul#adminmenu > li.current > a.current:after {
 /* Themes */
 
 .theme-browser .theme.active .theme-name,
-.theme-browser .theme.add-new-theme:hover:after {
+.theme-browser .theme.add-new-theme a:hover:after,
+.theme-browser .theme.add-new-theme a:focus:after {
 	background: $highlight-color;
 }
 
-.theme-browser .theme.add-new-theme:hover span:after {
+.theme-browser .theme.add-new-theme a:hover span:after,
```

Comment:
Begin pseudo elements with double colons: `::`<br>Selector should have depth of applicability no greater than 2, but was 4